<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=description content="Webpage of the Resendis Lab"><meta name=viewport content="width=device-width,initial-scale=1,minimum-scale=1"><meta name=generator content="Hugo 0.79.0"><title>Webpage of the Resendis Lab</title><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Lato:300,400,500,700" type=text/css><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel=stylesheet><link rel=stylesheet href=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.indigo-green.min.css><link rel=stylesheet href=../css/styles.css><link rel=icon type=image/png href=../media/favicon.png></head><body><div class="mdl-layout mdl-js-layout mdl-layout--fixed-drawer mdl-layout--fixed-header"><header class="mdl-layout__header mdl-layout--small-screen-only"></header><div class=mdl-layout__drawer><a class=mdl-layout-title href=../><img src=../media/logo.png width=80% align=middle style=margin-top:2em></a><nav class=mdl-navigation><a class=mdl-navigation__link href=../>Home</a>
<a class=mdl-navigation__link href=../about>About</a>
<a class=mdl-navigation__link href=../members>Members</a>
<a class=mdl-navigation__link href=../positions>Work with us</a>
<a class=mdl-navigation__link href=../pubs>Publications</a>
<a class=mdl-navigation__link href=../projects>Projects</a>
<a class=mdl-navigation__link href=../software>Software</a>
<a class=mdl-navigation__link href=../events>Events</a>
<a class=mdl-navigation__link href=../about/#directions>Contact</a></nav></div><main class="mdl-layout__content mdl-color--grey-100"><div class="page-content mdl-grid"><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet" style=margin-bottom:2em><div class="card mdl-card mdl-shadow--4dp"><div class="mdl-card__title mdl-color--indigo-900" style=height:auto><h1 class=mdl-card__title-text>Projects</h1></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>In silico study of metabolic reprogramming during epithelial-mesenchymal transition</h1><h2 class=mdl-color-text--teal-500>Meztli Matadamas</h2><br><p>An epithelial-mesenchymal transition (EMT) is a biologic process that allows a polarized epithelial cell, which normally interacts with basement membrane via its basal surface, to undergo multiple biochemical changes that enable it to assume a mesenchymal cell phenotype, which includes enhanced migratory capacity, invasiveness, elevated resistance to apoptosis, and greatly increased production of ECM components. EMT induces invasive properties in epithelial tumors and promotes metastasis. Although EMT-mediated cellular and molecular changes are well understood, very little is known about EMT-induced metabolic changes.</p><p>The project combine high-throughput data to understand metabolic changes before and after EMT in lung cancer cell lines. In particular to find main fluxes used during EMT. To that extent we employ methods from bioinformatics and Systems Biology. Our goal is to found specific targets which could stop or reverse EMT in cancer cells.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Microbiome metabolism and diabetes</h1><h2 class=mdl-color-text--teal-500>Jean Paul Sanchez</h2><br><p>Alterations in the microbiome has been associated with diabetes progression.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Immunology and cancer: Boolean Modeling of regulatory networks</h1><h2 class=mdl-color-text--teal-500>Ugo Avila Ponce de Leon</h2><br><p>Macrophages are cells of the innate immune system endowed with the capacity to orchestrate the immune response in human tissues. Due to their plasticity biological property, they polarize to several subtypes based on the actions of the tumor microenvironment. These cells have plasticity, because once they are committed to a subtype fate, they can polarize to another subtype by simply modifying the microenvironment. We integrated experimental data for the construction of a network that will explain the plasticity and the importance of the microenvironment in shaping the polarization of macrophages. The mathematical model was used to describe the genetic control points of macrophage polarization and plasticity, and it can function as groundwork and guidance for an immunotherapeutic approach to modulate the proliferation of cancer cells.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Integrating transcriptomic and metabolomic to understand hepatocellular carcinoma in a rat model</h1><h2 class=mdl-color-text--teal-500>Erika Hernandez</h2><br><p>Hepatocellular carcinoma (HCC) is now the third leading cause of cancer deaths worldwide, with over 500,000 people affected. It occurs predominantly in patients with underlying chronic liver disease and cirrhosis. Despite this, knowledge about the metabolic states of this disease is limited.
Using a rat model that recreates some of the most important characteristics of HCC, including cirrhosis, we aim to understand the metabolic state when compared to healthy liver. To this end we will integrate transcriptomic and metabolic data in a systems biology framework that point us changes in reactions.
This data would not only helped us identify reactions important to maintain the cancerous state but also help us survey the regulatory mechanism this is achieved.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Metabolic heterogeneity in cancer and its applications in Personalized Medicine</h1><h2 class=mdl-color-text--teal-500>Christian Diener</h2><br><p>Cancer is a very heterogeneous disease and tumors can differ greatly across and within different cancer types. Consequently, cancer is not a single disease but thousands. One property shared by all cancers is the ability to sustain chronic uncontrolled proliferation which raises the question how different cancers alter their metabolism in order to achieve consistent proliferation.</p><p>In this project we combine large-scale genomic data from DNA and RNA sequencing as well as proteomics and metabolomics to understand the connection between variations in the genotype and cancer metabolism. In particular we are asking the question whether distinct genomic aberrations such as mutations or changes in transcription can be related to respective changes in cellular metabolism. To that extent we employ methods from Systems Biology as well as from Data Science and Machine Learning in order to connect genetic information to specific metabolic phenotypes.</p><p>Our aim is to use the knowledge we gain in the context of personalized medicine, particularly the use of genotyping for the prediction of the best course of treatment for a specific patient.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Systems biology and bioinformatics of single cell RNAseq data.</h1><h2 class=mdl-color-text--teal-500>Thelma Escobedo</h2><br><p>Research in personalized therapy has taken relevance because treatment failures due to intratumoral heterogenety which refers to celular diversity or subpopulations
forming within the tumor. Currently, given complex molecular processes of cancer there has been greater use of omic technologies and computational analysis. With the purpuse to
contribute in this line, we have opened a new line of research to describe the progress of expression profiles in tumor cell lines through bioinformatic analysis of single cell
RNAseq data. Likewise, this work will contribute to infer the principles that guide the population heterogeneity mechanisms for the design of new optimized strategies for the
treatment of cancer.</p></div></div></div><div class="mdl-cell mdl-cell--12-col-desktop mdl-cell--8-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>The impact of the microRNAs in the metabolic reprogramming of the MCF-7 cells during the spheroids development</h1><h2 class=mdl-color-text--teal-500>Erick Muci√±o</h2><br><p>Alterations in the metabolism are a common property in cancer cells, so that, many efforts have been directed to develop models to understand the mechanism by which cancer cells behave differently compared to normal tissues. In recent years, it has been reported that microRNAs (miRNAs) are involved in the regulation of all biological process, and there are evidences that shown its dysregulation play an important role in the development and progression of cancer. Hence, generate models that allow the integration of miRNAs regulation in cancer metabolism will allow us to analyze in a systematically and systemic manner the relations and potential mechanism underlying between miRNAs and central pathways of metabolism.</p><p>It is imperative to know the mechanism governing the pathogenesis and progression of cancer to design therapies with greater impact on diagnosis and disease progression. So, this project aims to suggest mechanism that trigger the metabolic change in a breast cancer cell line (MCF-7), integrating miRNAs network. To this end, we propose to develop a scheme of systems biology, which allow us to make an integrative analysis of regulatory networks of miRNAs and metabolism in MCF-7. This approach will allow us to develop models capable of identifying potential therapeutic targets with greater impact, biomarkers that allow early detection of cancer and penetrate in global mechanism in clinical cases.</p></div></div></div></div></main></div><script src=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.min.js></script></body></html>