<!doctype html><html lang=en><head><meta charset=utf-8><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=description content="Webpage of the Resendis Lab"><meta name=viewport content="width=device-width,initial-scale=1,minimum-scale=1"><meta name=generator content="Hugo 0.79.0"><title>Webpage of the Resendis Lab</title><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Lato:300,400,500,700" type=text/css><link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel=stylesheet><link rel=stylesheet href=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.indigo-green.min.css><link rel=stylesheet href=../css/styles.css><link rel=icon type=image/png href=../media/favicon.png></head><body><div class="mdl-layout mdl-js-layout mdl-layout--fixed-drawer mdl-layout--fixed-header"><header class="mdl-layout__header mdl-layout--small-screen-only"></header><div class=mdl-layout__drawer><a class=mdl-layout-title href=../><img src=../media/logo.png width=80% align=middle style=margin-top:2em></a><nav class=mdl-navigation><a class=mdl-navigation__link href=../>Home</a>
<a class=mdl-navigation__link href=../about>About</a>
<a class=mdl-navigation__link href=../members>Members</a>
<a class=mdl-navigation__link href=../positions>Work with us</a>
<a class=mdl-navigation__link href=../pubs>Publications</a>
<a class=mdl-navigation__link href=../projects>Projects</a>
<a class=mdl-navigation__link href=../software>Software</a>
<a class=mdl-navigation__link href=../events>Events</a>
<a class=mdl-navigation__link href=../about/#directions>Contact</a></nav></div><main class="mdl-layout__content mdl-color--grey-100"><div class="page-content mdl-grid"><div class="mdl-tabs mdl-js-tabs mdl-js-ripple-effect"><div class=mdl-tabs__tab-bar><a href=#articles class="mdl-tabs__tab is-active">Articles</a>
<a href=#books class=mdl-tabs__tab>Books</a></div><div class="mdl-tabs__panel is-active" id=articles><br><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>MicroRNAs Regulate Metabolic Phenotypes During Multicellular Tumor Spheroids Progression</h1><h3 class=mdl-color-text--gray>Frontiers in Oncology&nbsp;2020</h3><h2 class=mdl-color-text--teal-500>Erick Andrés Muciño-Olmos, Aarón Vázquez-Jiménez, Diana Elena López-Esparza, Vilma Maldonado, Mahara Valverde and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>During tumor progression, cancer cells ire their metabolism to face their bioenergetic
demands. In recent years, microRNAs (miRNAs) have emerged as regulatory elements that
inhibit the translation and stability of crucial mRNAs, some of them causing direct
metabolic alterations in cancer. In this study, we investigated the relationship between
miRNAs and their targets mRNAs that control metabolism, and how this fine-tuned
regulation is diversified depending on the tumor stage. To do so, we implemented a
paired analysis of RNA-seq and small RNA-seq in a breast cancer cell line (MCF7). The
cell line was cultured in multicellular tumor spheroid (MCTS) and monoculture
conditions. For MCTS, we selected two-time points during their development to
recapitulate a proliferative and quiescent stage and contrast their miRNA and mRNA
expression patterns associated with metabolism. As a result, we identified a set of new
direct putative regulatory interactions between miRNAs and metabolic mRNAs
representative for proliferative and quiescent stages. Notably, our study allows us to
suggest that miR-3143 regulates the carbon metabolism by targeting hexokinase-2. Also,
we found that the overexpression of several miRNAs could directly overturn the
expression of mRNAs that control glycerophospholipid and N-Glycan metabolism. While this
set of miRNAs downregulates their expression in the quiescent stage, the same set is
upregulated in proliferative stages. This last finding suggests an additional metabolic
switch of the above mentioned metabolic pathways between the quiescent and proliferative
stages. Our results contribute to a better understanding of how miRNAs modulate the
metabolic landscape in breast cancer MCTS, which eventually will help to design new
strategies to mitigate cancer phenotype.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fonc.2020.582396>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Analysis of Epithelial-Mesenchymal Transition Metabolism Identifies Possible Cancer Biomarkers Useful in Diverse Genetic Backgrounds</h1><h3 class=mdl-color-text--gray>Frontiers in Onology&nbsp;2020</h3><h2 class=mdl-color-text--teal-500>Meztli Matadamas-Guzman, Cecilia Zazueta, Emilio Rojas and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>Epithelial-to-mesenchymal transition (EMT) relates to many molecular and cellular alterations that occur when epithelial cells undergo a switch in differentiation generating mesenchymal-like cells with newly acquired migratory and invasive properties. In cancer cells, EMT leads to drug resistance and metastasis. Moreover, differences in genetic backgrounds, even between patients with the same type of cancer, also determine resistance to some treatments. Metabolic rewiring is essential to induce EMT, hence it is important to identify key metabolic elements for this process, which can be later used to treat cancer cells with different genetic backgrounds. Here we used a mathematical modeling approach to determine which are the metabolic reactions altered after induction of EMT, based on metabolomic and transcriptional data of three non-small cell lung cancer (NSCLC) cell lines. The model suggested that the most affected pathways were the Krebs cycle, amino acid metabolism, and glutathione metabolism. However, glutathione metabolism had many alterations either on the metabolic reactions or at the transcriptional level in the three cell lines. We identified Glutamate-cysteine ligase (GCL), a key enzyme of glutathione synthesis, as an important common feature that is dysregulated after EMT. Analyzing survival data of men with lung cancer, we observed that patients with mutations in GCL catalytic subunit (GCLC) or Glutathione peroxidase 1 (GPX1) genes survived less time than people without mutations on these genes. Besides, patients with low expression of ANPEP, GPX3 and GLS genes also survived less time than those with high expression. Hence, we propose that glutathione metabolism and glutathione itself could be good targets to delay or potentially prevent EMT induction in NSCLC cell lines.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fonc.2020.01309>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/32850411>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Unveiling functional heterogeneity in breast cancer multicellular tumor spheroids through single-cell RNA-seq</h1><h3 class=mdl-color-text--gray>Scientific Reports&nbsp;2020</h3><h2 class=mdl-color-text--teal-500>Erick Andrés Muciño-Olmos, Aarón Vázquez-Jiménez, Ugo Avila-Ponce de León, Meztli Matadamas-Guzman, Vilma Maldonado, Tayde López-Santaella, Abrahan Hernández-Hernández and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>Heterogeneity is an intrinsic characteristic of cancer. Even in isogenic tumors, cell populations exhibit differential cellular programs that overall supply malignancy and decrease treatment efficiency. In this study, we investigated the functional relationship among cell subtypes and how this interdependency can promote tumor development in a cancer cell line. To do so, we performed single-cell RNA-seq of MCF7 Multicellular Tumor Spheroids as a tumor model. Analysis of single-cell transcriptomes at two-time points of the spheroid growth, allowed us to dissect their functional relationship. As a result, three major robust cellular clusters, with a non-redundant complementary composition, were found. Meanwhile, one cluster promotes proliferation, others mainly activate mechanisms to invade other tissues and serve as a reservoir population conserved over time. Our results provide evidence to see cancer as a systemic unit that has cell populations with task stratification with the ultimate goal of preserving the hallmarks in tumors.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1038/s41598-020-69026-7>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/32728097>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Memote: A community driven effort towards a standardized genome-scale metabolic model test suite</h1><h3 class=mdl-color-text--gray>Nature Biotechnology&nbsp;2020</h3><h2 class=mdl-color-text--teal-500>Christian Lieven, Moritz E. Beber, Brett G. Olivier, Frank T. Bergmann, Meric Ataman, Parizad Babaei, Jennifer A. Bartell, Lars M. Blank, Siddharth Chauhan, Kevin Correia, Christian Diener, Andreas Dräger, Birgitta E. Ebert, Janaka N. Edirisinghe, Jose P. Faria, Adam Feist, Georgios Fengos, Ronan M. T. Fleming, Beatriz García-Jiménez, Vassily Hatzimanikatis, Wout van Helvoirt, Christopher S. Henry, Henning Hermjakob, Markus J. Herrgård, Hyun Uk Kim, Zachary King, Jasper J. Koehorst, Steffen Klamt, Edda Klipp, Meiyappan Lakshmanan, Nicolas Le Novère, Dong-Yup Lee, Sang Yup Lee, Sunjae Lee, Nathan E. Lewis, Hongwu Ma, Daniel Machado, Radhakrishnan Mahadevan, Paulo Maia, Adil Mardinoglu, Gregory L. Medlock, Jonathan M. Monk, Jens Nielsen, Lars Keld Nielsen, Juan Nogales, Intawat Nookaew, Osbaldo Resendis-Antonio, Bernhard O. Palsson, Jason A. Papin, Kiran R. Patil, Mark Poolman, Nathan D. Price, Anne Richelle, Isabel Rocha, Benjamin J. Sanchez, Peter J. Schaap, Rahuman S. Malik Sheriff, Saeed Shoaie, Nikolaus Sonnenschein, Bas Teusink, Paulo Vilaça, Jon Olav Vik, Judith A. Wodke, Joana C. Xavier, Qianqian Yuan, Maksim Zakhartsev and Cheng Zhang</h2><p style=font-size:13px;margin-top:.5em></p><p>Several studies have shown that neither the formal representation nor the functional requirements of genome-scale metabolic models (GEMs) are precisely defined. Without a consistent standard, comparability, reproducibility, and interoperability of models across groups and software tools cannot be guaranteed.
Here, we present memote (<a href=https://github.com/opencobra/memote>https://github.com/opencobra/memote</a>) an open-source software containing a community-maintained, standardized set of metabolic model tests. The tests cover a range of aspects from annotations to conceptual integrity and can be extended to include experimental datasets for automatic model validation. In addition to testing a model once, memote can be configured to do so automatically, i.e., while building a GEM. A comprehensive report displays the model’s performance parameters, which supports informed model development and facilitates error detection.
Memote provides a measure for model quality that is consistent across reconstruction platforms and analysis software and simplifies collaboration within the community by establishing workflows for publicly hosted and version controlled models.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1101/350991>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/32123384>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Micom: metagenome-scale modeling to infer metabolic interactions in the microbiota.</h1><h3 class=mdl-color-text--gray>Msystems&nbsp;2020</h3><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>Alterations in the gut microbiota have been associated with a variety of medical conditions such as obesity, Crohn&rsquo;s disease and diabetes. However, establishing the causality between the microbial composition and disease remains a challenge. We introduce a strategy based on metabolic models of complete microbial gut communities in order to derive the particular metabolic consequences of the microbial composition for the diabetic gut in a balanced cohort of 186 individuals. By using a heuristic optimization approach based on L2 regularization we were able to obtain a unique set of realistic growth rates that allows growth for the majority of observed taxa in a sample. We also integrated various additional constraints such as diet and the measured abundances of microbial species to derive the resulting metabolic alterations for individual metagenomic samples. In particular, we show that growth rates vary greatly across samples and that there exists a network of bacteria implicated in health and disease that mutually influence each others growth rates. Studying individual exchange fluxes between the microbiota and the gut lumen we observed that consumption of metabolites by the microbiota follows a niche structure whereas production of short chain fatty acids by the microbiota was highly sample-specific and was altered in type 2 diabetes and restored after metformin treatment in samples from danish individuals. Additionally, we found that production of butyrate could not be easily influenced by single-target interventions.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/%2010.1128/mSystems.00606-19>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/31964767>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Distinct microbes, metabolites, and ecologies define the microbiome in deficient and proficient mismatch repair colorectal cancers.</h1><h3 class=mdl-color-text--gray>Genome Medicine&nbsp;2018</h3><h2 class=mdl-color-text--teal-500>Vanessa L. Hale, Patricio Jeraldo, Jun Chen, Michael Mundy, Janet Yao, Sambhawa Priya, Gary Keeney, Kelly Lyke, Jason Ridlon, Bryan A. White, Amy J. French, Stephen N. Thibodeau, Christian Diener, Osbaldo Resendis-Antonio, Jaime Gransee, Tumpa Dutta, Xuan-Mai Petterson, Jaeyun Sung, Ran Blekhman, Lisa Boardman, David Larson, Heidi Nelson and Nicholas Chia</h2><p style=font-size:13px;margin-top:.5em></p><p>Links between colorectal cancer (CRC) and the gut microbiome have been established, but the specific microbial species and their role in carcinogenesis remain an active area of inquiry. Our understanding would be enhanced by better accounting for tumor subtype, microbial community interactions, metabolism, and ecology. We collected paired colon tumor and normal-adjacent tissue and mucosa samples from 83 individuals who underwent partial or total colectomies for CRC. Mismatch repair (MMR) status was determined in each tumor sample and classified as either deficient MMR (dMMR) or proficient MMR (pMMR) tumor subtypes. Samples underwent 16S rRNA gene sequencing and a subset of samples from 50 individuals were submitted for targeted metabolomic analysis to quantify amino acids and short-chain fatty acids. A PERMANOVA was used to identify the biological variables that explained variance within the microbial communities. dMMR and pMMR microbial communities were then analyzed separately using a generalized linear mixed effects model that accounted for MMR status, sample location, intra-subject variability, and read depth. Genome-scale metabolic models were then used to generate microbial interaction networks for dMMR and pMMR microbial communities. We assessed global network properties as well as the metabolic influence of each microbe within the dMMR and pMMR networks.We demonstrate distinct roles for microbes in dMMR and pMMR CRC. Bacteroides fragilis and sulfidogenic Fusobacterium nucleatum were significantly enriched in dMMR CRC, but not pMMR CRC. These findings were further supported by metabolic modeling and metabolomics indicating suppression of B. fragilis in pMMR CRC and increased production of amino acid proxies for hydrogen sulfide in dMMR CRC.Integrating tumor biology and microbial ecology highlighted distinct microbial, metabolic, and ecological properties unique to dMMR and pMMR CRC. This approach could critically improve our ability to define, predict, prevent, and treat colorectal cancers.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1186/s13073-018-0586-6>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/30376889>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Synthesis of multi-omic data and community metabolic models reveals insights into the role of hydrogen sulfide in colon cancer.</h1><h3 class=mdl-color-text--gray>Methods (San Diego, Calif.)&nbsp;2018</h3><h2 class=mdl-color-text--teal-500>Vanessa L Hale, Patricio Jeraldo, Michael Mundy, Janet Yao, Gary Keeney, Nancy Scott, E Heidi Cheek, Jennifer Davidson, Megan Green, Christine Martinez, John Lehman, Chandra Pettry, Erica Reed, Kelly Lyke, Bryan A White, Christian Diener, Osbaldo Resendis-Antonio, Jaime Gransee, Tumpa Dutta, Xuan-Mai Petterson, Lisa Boardman, David Larson, Heidi Nelson and Nicholas Chia</h2><p style=font-size:13px;margin-top:.5em></p><p>Multi-omic data and genome-scale microbial metabolic models have allowed us to examine microbial communities, community function, and interactions in ways that were not available to us historically. Now, one of our biggest challenges is determining how to integrate data and maximize data potential. Our study demonstrates one way in which to test a hypothesis by combining multi-omic data and community metabolic models. Specifically, we assess hydrogen sulfide production in colorectal cancer based on stool, mucosa, and tissue samples collected on and off the tumor site within the same individuals. 16S rRNA microbial community and abundance data were used to select and inform the metabolic models. We then used MICOM, an open source platform, to track the metabolic flux of hydrogen sulfide through a defined microbial community that either represented on-tumor or off-tumor sample communities. We also performed targeted and untargeted metabolomics, and used the former to quantitatively evaluate our model predictions. A deeper look at the models identified several unexpected but feasible reactions, microbes, and microbial interactions involved in hydrogen sulfide production for which our 16S and metabolomic data could not account. These results will guide future in vitro, in vivo, and in silico tests to establish why hydrogen sulfide production is increased in tumor tissue.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1016/j.ymeth.2018.04.024>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/29704665>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Editorial: Systems Biology and the Challenge of Deciphering the Metabolic Mechanisms Underlying Cancer</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2017</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio and Christian Diener</h2><p style=font-size:13px;margin-top:.5em></p><p>This is the Editorial for our Frontiers Research Topic &ldquo;Systems Biology and
the challenge of deciphering the metabolic mechanisms underlying cancer&rdquo;.</p><p>The corresponding E-Book will be available soon.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2017.00537>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>"Gestaltomics": Systems Biology Schemes for the Study of Neuropsychiatric Diseases.</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2017</h3><h2 class=mdl-color-text--teal-500>Nora A Gutierrez Najera, Osbaldo Resendis-Antonio and Humberto Nicolini</h2><p style=font-size:13px;margin-top:.5em>Keywords: diagnosis, lung cancer, omics, psychiatry, systems biology</p><p>The integration of different sources of biological information about what defines a behavioral phenotype is difficult to unify in an entity that reflects the arithmetic sum of its individual parts. In this sense, the challenge of Systems Biology for understanding the &ldquo;psychiatric phenotype&rdquo; is to provide an improved vision of the shape of the phenotype as it is visualized by &ldquo;Gestalt&rdquo; psychology, whose fundamental axiom is that the observed phenotype (behavior or mental disorder) will be the result of the integrative composition of every part. Therefore, we propose the term &ldquo;Gestaltomics&rdquo; as a term from Systems Biology to integrate data coming from different sources of information (such as the genome, transcriptome, proteome, epigenome, metabolome, phenome, and microbiome). In addition to this biological complexity, the mind is integrated through multiple brain functions that receive and process complex information through channels and perception networks (i.e., sight, ear, smell, memory, and attention) that in turn are programmed by genes and influenced by environmental processes (epigenetic). Today, the approach of medical research in human diseases is to isolate one disease for study; however, the presence of an additional disease (co-morbidity) or more than one disease (multimorbidity) adds complexity to the study of these conditions. This review will present the challenge of integrating psychiatric disorders at different levels of information (Gestaltomics). The implications of increasing the level of complexity, for example, studying the co-morbidity with another disease such as cancer, will also be discussed.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2017.00286>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/28536537>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Natural selection drove metabolic specialization of the chromatophore in Paulinella chromatophora.</h1><h3 class=mdl-color-text--gray>BMC Evolutionary Biology&nbsp;2017</h3><h2 class=mdl-color-text--teal-500>Cecilio Valadez-Cano, Roberto Olivares-Hernández, Osbaldo Resendis-Antonio, Alexander DeLuna and Luis Delaye</h2><p style=font-size:13px;margin-top:.5em>Keywords: Adaptation, Endosymbiont, Metabolic evolution, Metabolic integration</p><p>Genome degradation of host-restricted mutualistic endosymbionts has been attributed to inactivating mutations and genetic drift while genes coding for host-relevant functions are conserved by purifying selection. Unlike their free-living relatives, the metabolism of mutualistic endosymbionts and endosymbiont-originated organelles is specialized in the production of metabolites which are released to the host. This specialization suggests that natural selection crafted these metabolic adaptations. In this work, we analyzed the evolution of the metabolism of the chromatophore of Paulinella chromatophora by in silico modeling. We asked whether genome reduction is driven by metabolic engineering strategies resulted from the interaction with the host. As its widely known, the loss of enzyme coding genes leads to metabolic network restructuring sometimes improving the production rates. In this case, the production rate of reduced-carbon in the metabolism of the chromatophore.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1186/s12862-017-0947-6>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/28410570>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Personalized Prediction of Proliferation Rates and Metabolic Liabilities in Cancer Biopsies.</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2017</h3><h2 class=mdl-color-text--teal-500>Christian Diener and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em>Keywords: NCI60, TCGA, flux balance analysis, personalized medicine, proliferation, systems biology</p><p>Cancer is a heterogeneous disease and its genetic and metabolic mechanism may manifest differently in each patient. This creates a demand for studies that can characterize phenotypic traits of cancer on a per-sample basis. Combining two large data sets, the NCI60 cancer cell line panel, and The Cancer Genome Atlas, we used a linear interaction model to predict proliferation rates for more than 12,000 cancer samples across 33 different cancers from The Cancer Genome Atlas. The predicted proliferation rates are associated with patient survival and cancer stage and show a strong heterogeneity in proliferative capacity within and across different cancer panels. We also show how the obtained proliferation rates can be incorporated into genome-scale metabolic reconstructions to obtain the metabolic fluxes for more than 3000 cancer samples that identified specific metabolic liabilities for nine cancer panels. Here we found that affected pathways coincided with the literature, with pentose phosphate pathway, retinol, and branched-chain amino acid metabolism being the most panel-specific alterations and fatty acid metabolism and ROS detoxification showing homogeneous metabolic activities across all cancer panels. The presented strategy has potential applications in personalized medicine since it can leverage gene expression signatures for cell line based prediction of additional metabolic properties which might help in constraining personalized metabolic models and improve the identification of metabolic alterations in cancer for individual patients.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2016.00644>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/28082911>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Host-Microbiome Interaction and Cancer: Potential Application in Precision Medicine.</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2016</h3><h2 class=mdl-color-text--teal-500>Alejandra V Contreras, Benjamin Cocom-Chan, Georgina Hernandez-Montes, Tobias Portillo-Bobadilla and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em>Keywords: cancer metabolism, metabolome, microbiome, next generation sequencing (NGS), precision medicine, systems integration</p><p>It has been experimentally shown that host-microbial interaction plays a major role in shaping the wellness or disease of the human body. Microorganisms coexisting in human tissues provide a variety of benefits that contribute to proper functional activity in the host through the modulation of fundamental processes such as signal transduction, immunity and metabolism. The unbalance of this microbial profile, or dysbiosis, has been correlated with the genesis and evolution of complex diseases such as cancer. Although this latter disease has been thoroughly studied using different high-throughput (HT) technologies, its heterogeneous nature makes its understanding and proper treatment in patients a remaining challenge in clinical settings. Notably, given the outstanding role of host-microbiome interactions, the ecological interactions with microorganisms have become a new significant aspect in the systems that can contribute to the diagnosis and potential treatment of solid cancers. As a part of expanding precision medicine in the area of cancer research, efforts aimed at effective treatments for various kinds of cancer based on the knowledge of genetics, biology of the disease and host-microbiome interactions might improve the prediction of disease risk and implement potential microbiota-directed therapeutics. In this review, we present the state of the art of sequencing and metabolome technologies, computational methods and schemes in systems biology that have addressed recent breakthroughs of uncovering relationships or associations between microorganisms and cancer. Together, microbiome studies extend the horizon of new personalized treatments against cancer from the perspective of precision medicine through a synergistic strategy integrating clinical knowledge, HT data, bioinformatics, and systems biology.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2016.00606>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/28018236>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Evolution of Centrality Measurements for the Detection of Essential Proteins in Biological Networks.</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2016</h3><h2 class=mdl-color-text--teal-500>Mahdi Jalili, Ali Salehzadeh-Yazdi, Shailendra Gupta, Olaf Wolkenhauer, Marjan Yaghmaie, Osbaldo Resendis-Antonio and Kamran Alimoghaddam</h2><p style=font-size:13px;margin-top:.5em>Keywords: biological centrality, biological network, centrality, essentiality, topological network analysis</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2016.00375>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/27616995>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>The space of enzyme regulation in HeLa cells can be inferred from its intracellular metabolome.</h1><h3 class=mdl-color-text--gray>Scientific Reports&nbsp;2016</h3><h2 class=mdl-color-text--teal-500>Christian Diener, Felipe Muñoz-Gonzalez, Sergio Encarnación and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>During the transition from a healthy state to a cancerous one, cells alter their metabolism to increase proliferation. The underlying metabolic alterations may be caused by a variety of different regulatory events on the transcriptional or post-transcriptional level whose identification contributes to the rational design of therapeutic targets. We present a mechanistic strategy capable of inferring enzymatic regulation from intracellular metabolome measurements that is independent of the actual mechanism of regulation. Here, enzyme activities are expressed by the space of all feasible kinetic constants (k-cone) such that the alteration between two phenotypes is given by their corresponding kinetic spaces. Deriving an expression for the transformation of the healthy to the cancer k-cone we identified putative regulated enzymes between the HeLa and HaCaT cell lines. We show that only a few enzymatic activities change between those two cell lines and that this regulation does not depend on gene transcription but is instead post-transcriptional. Here, we identify phosphofructokinase as the major driver of proliferation in HeLa cells and suggest an optional regulatory program, associated with oxidative stress, that affects the activity of the pentose phosphate pathway.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1038/srep28415>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/27335086>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Modeling metabolism: a window toward a comprehensive interpretation of networks in cancer.</h1><h3 class=mdl-color-text--gray>Seminars in Cancer Biology&nbsp;2014</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Carolina González-Torres, Gustavo Jaime-Muñoz, Claudia Erika Hernandez-Patiño and Carlos Felipe Salgado-Muñoz</h2><p style=font-size:13px;margin-top:.5em>Keywords: Cancer metabolism, Mathematical models, P4 medicine, Systems biology: Constraint-based modeling</p><p>Given the multi-factorial nature of cancer, uncovering its metabolic alterations and evaluating their implications is a major challenge in biomedical sciences that will help in the optimal design of personalized treatments. The advance of high-throughput technologies opens an invaluable opportunity to monitor the activity at diverse biological levels and elucidate how cancer originates, evolves and responds under drug treatments. To this end, researchers are confronted with two fundamental questions: how to interpret high-throughput data and how this information can contribute to the development of personalized treatment in patients. A variety of schemes in systems biology have been suggested to characterize the phenotypic states associated with cancer by utilizing computational modeling and high-throughput data. These theoretical schemes are distinguished by the level of complexity of the biological mechanisms that they represent and by the computational approaches used to simulate them. Notably, these theoretical approaches have provided a proper framework to explore some distinctive metabolic mechanisms observed in cancer cells such as the Warburg effect. In this review, we focus on presenting a general view of some of these approaches whose application and integration will be crucial in the transition from local to global conclusions in cancer studies. We are convinced that multidisciplinary approaches are required to construct the bases of an integrative and personalized medicine, which has been and remains a fundamental task in the medicine of this century.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1016/j.semcancer.2014.04.003>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/24747697>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Systems biology of cancer: moving toward the integrative study of the metabolic alterations in cancer cells.</h1><h3 class=mdl-color-text--gray>Frontiers in Physiology&nbsp;2013</h3><h2 class=mdl-color-text--teal-500>Claudia E Hernández Patiño, Gustavo Jaime-Muñoz and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em>Keywords: cancer metabolic phenotype, computational modeling of metabolism, constraint-based modeling, genome scale metabolic reconstruction, high throughput biology</p><p>One of the main objectives in systems biology is to understand the biological mechanisms that give rise to the phenotype of a microorganism by using high-throughput technologies (HTs) and genome-scale mathematical modeling. The computational modeling of genome-scale metabolic reconstructions is one systemic and quantitative strategy for characterizing the metabolic phenotype associated with human diseases and potentially for designing drugs with optimal clinical effects. The purpose of this short review is to describe how computational modeling, including the specific case of constraint-based modeling, can be used to explore, characterize, and predict the metabolic capacities that distinguish the metabolic phenotype of cancer cell lines. As we show herein, this computational framework is far from a pure theoretical description, and to ensure proper biological interpretation, it is necessary to integrate high-throughput data and generate predictions for later experimental assessment. Hence, genome-scale modeling serves as a platform for the following: (1) the integration of data from HTs, (2) the assessment of how metabolic activity is related to phenotype in cancer cell lines, and (3) the design of new experiments to evaluate the outcomes of the in silico analysis. By combining the functions described above, we show that computational modeling is a useful methodology to construct an integrative, systemic, and quantitative scheme for understanding the metabolic profiles of cancer cell lines, a first step to determine the metabolic mechanism by which cancer cells maintain and support their malignant phenotype in human tissues.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.3389/fphys.2012.00481>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/23316163>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Functional modules, structural topology, and optimal activity in metabolic networks.</h1><h3 class=mdl-color-text--gray>PLoS Computational Biology&nbsp;2012</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Magdalena Hernández, Yolanda Mora and Sergio Encarnación</h2><p style=font-size:13px;margin-top:.5em></p><p>Modular organization in biological networks has been suggested as a natural mechanism by which a cell coordinates its metabolic strategies for evolving and responding to environmental perturbations. To understand how this occurs, there is a need for developing computational schemes that contribute to integration of genomic-scale information and assist investigators in formulating biological hypotheses in a quantitative and systematic fashion. In this work, we combined metabolome data and constraint-based modeling to elucidate the relationships among structural modules, functional organization, and the optimal metabolic phenotype of Rhizobium etli, a bacterium that fixes nitrogen in symbiosis with Phaseolus vulgaris. To experimentally characterize the metabolic phenotype of this microorganism, we obtained the metabolic profile of 220 metabolites at two physiological stages: under free-living conditions, and during nitrogen fixation with P. vulgaris. By integrating these data into a constraint-based model, we built a refined computational platform with the capability to survey the metabolic activity underlying nitrogen fixation in R. etli. Topological analysis of the metabolic reconstruction led us to identify modular structures with functional activities. Consistent with modular activity in metabolism, we found that most of the metabolites experimentally detected in each module simultaneously increased their relative abundances during nitrogen fixation. In this work, we explore the relationships among topology, biological function, and optimal activity in the metabolism of R. etli through an integrative analysis based on modeling and metabolome data. Our findings suggest that the metabolic activity during nitrogen fixation is supported by interacting structural modules that correlate with three functional classifications: nucleic acids, peptides, and lipids. More fundamentally, we supply evidence that such modular organization during functional nitrogen fixation is a robust property under different environmental conditions.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1371/journal.pcbi.1002720>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/23071431>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Systems biology of bacterial nitrogen fixation: high-throughput technology and its integrative description with constraint-based modeling.</h1><h3 class=mdl-color-text--gray>BMC Systems Biology&nbsp;2011</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Magdalena Hernández, Emmanuel Salazar, Sandra Contreras, Gabriel Martínez Batallar, Yolanda Mora and Sergio Encarnación</h2><p style=font-size:13px;margin-top:.5em></p><p>Bacterial nitrogen fixation is the biological process by which atmospheric nitrogen is uptaken by bacteroids located in plant root nodules and converted into ammonium through the enzymatic activity of nitrogenase. In practice, this biological process serves as a natural form of fertilization and its optimization has significant implications in sustainable agricultural programs. Currently, the advent of high-throughput technology supplies with valuable data that contribute to understanding the metabolic activity during bacterial nitrogen fixation. This undertaking is not trivial, and the development of computational methods useful in accomplishing an integrative, descriptive and predictive framework is a crucial issue to decoding the principles that regulated the metabolic activity of this biological process.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1186/1752-0509-5-120>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/21801415>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Proteomic patterns of cervical cancer cell lines, a network perspective.</h1><h3 class=mdl-color-text--gray>BMC Systems Biology&nbsp;2011</h3><h2 class=mdl-color-text--teal-500>Juan Carlos Higareda-Almaraz, María del Rocío Enríquez-Gasca, Magdalena Hernández-Ortiz, Osbaldo Resendis-Antonio and Sergio Encarnación-Guevara</h2><p style=font-size:13px;margin-top:.5em></p><p>Cervical cancer is a major mortality factor in the female population. This neoplastic is an excellent model for studying the mechanisms involved in cancer maintenance, because the Human Papilloma Virus (HPV) is the etiology factor in most cases. With the purpose of characterizing the effects of malignant transformation in cellular activity, proteomic studies constitute a reliable way to monitor the biological alterations induced by this disease. In this contextual scheme, a systemic description that enables the identification of the common events between cell lines of different origins, is required to distinguish the essence of carcinogenesis.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1186/1752-0509-5-96>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/21696634>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Modeling core metabolism in cancer cells: surveying the topology underlying the Warburg effect.</h1><h3 class=mdl-color-text--gray>PloS One&nbsp;2010</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Alberto Checa and Sergio Encarnación</h2><p style=font-size:13px;margin-top:.5em></p><p>Alterations on glucose consumption and biosynthetic activity of amino acids, lipids and nucleotides are metabolic changes for sustaining cell proliferation in cancer cells. Irrevocable evidence of this fact is the Warburg effect which establishes that cancer cells prefers glycolysis over oxidative phosphorylation to generate ATP. Regulatory action over metabolic enzymes has opened a new window for designing more effective anti-cancer treatments. This enterprise is not trivial and the development of computational models that contribute to identifying potential enzymes for breaking the robustness of cancer cells is a priority.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1371/journal.pone.0012383>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/20811631>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Filling kinetic gaps: dynamic modeling of metabolism where detailed kinetic information is lacking.</h1><h3 class=mdl-color-text--gray>PloS One&nbsp;2009</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p>Integrative analysis between dynamical modeling of metabolic networks and data obtained from high throughput technology represents a worthy effort toward a holistic understanding of the link among phenotype and dynamical response. Even though the theoretical foundation for modeling metabolic network has been extensively treated elsewhere, the lack of kinetic information has limited the analysis in most of the cases. To overcome this constraint, we present and illustrate a new statistical approach that has two purposes: integrate high throughput data and survey the general dynamical mechanisms emerging for a slightly perturbed metabolic network.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1371/journal.pone.0004967>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/19305506>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Regulation by transcription factors in bacteria: beyond description.</h1><h3 class=mdl-color-text--gray>FEMS Microbiology Reviews&nbsp;2008</h3><h2 class=mdl-color-text--teal-500>Enrique Balleza, Lucia N López-Bojorquez, Agustino Martínez-Antonio, Osbaldo Resendis-Antonio, Irma Lozada-Chávez, Yalbi I Balderas-Martínez, Sergio Encarnación and Julio Collado-Vides</h2><p style=font-size:13px;margin-top:.5em></p><p>Transcription is an essential step in gene expression and its understanding has been one of the major interests in molecular and cellular biology. By precisely tuning gene expression, transcriptional regulation determines the molecular machinery for developmental plasticity, homeostasis and adaptation. In this review, we transmit the main ideas or concepts behind regulation by transcription factors and give just enough examples to sustain these main ideas, thus avoiding a classical ennumeration of facts. We review recent concepts and developments: cis elements and trans regulatory factors, chromosome organization and structure, transcriptional regulatory networks (TRNs) and transcriptomics. We also summarize new important discoveries that will probably affect the direction of research in gene regulation: epigenetics and stochasticity in transcriptional regulation, synthetic circuits and plasticity and evolution of TRNs. Many of the new discoveries in gene regulation are not extensively tested with wetlab approaches. Consequently, we review this broad area in Inference of TRNs and Dynamical Models of TRNs. Finally, we have stepped backwards to trace the origins of these modern concepts, synthesizing their history in a timeline schema.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1111/j.1574-6976.2008.00145.x>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/19076632>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Metabolic reconstruction and modeling of nitrogen fixation in Rhizobium etli.</h1><h3 class=mdl-color-text--gray>PLoS Computational Biology&nbsp;2007</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Jennifer L Reed, Sergio Encarnación, Julio Collado-Vides and Bernhard Ø Palsson</h2><p style=font-size:13px;margin-top:.5em></p><p>Rhizobiaceas are bacteria that fix nitrogen during symbiosis with plants. This symbiotic relationship is crucial for the nitrogen cycle, and understanding symbiotic mechanisms is a scientific challenge with direct applications in agronomy and plant development. Rhizobium etli is a bacteria which provides legumes with ammonia (among other chemical compounds), thereby stimulating plant growth. A genome-scale approach, integrating the biochemical information available for R. etli, constitutes an important step toward understanding the symbiotic relationship and its possible improvement. In this work we present a genome-scale metabolic reconstruction (iOR363) for R. etli CFN42, which includes 387 metabolic and transport reactions across 26 metabolic pathways. This model was used to analyze the physiological capabilities of R. etli during stages of nitrogen fixation. To study the physiological capacities in silico, an objective function was formulated to simulate symbiotic nitrogen fixation. Flux balance analysis (FBA) was performed, and the predicted active metabolic pathways agreed qualitatively with experimental observations. In addition, predictions for the effects of gene deletions during nitrogen fixation in Rhizobia in silico also agreed with reported experimental data. Overall, we present some evidence supporting that FBA of the reconstructed metabolic network for R. etli provides results that are in agreement with physiological observations. Thus, as for other organisms, the reconstructed genome-scale metabolic network provides an important framework which allows us to compare model predictions with experimental measurements and eventually generate hypotheses on ways to improve nitrogen fixation.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1371/journal.pcbi.0030192>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/17922569>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Identification of regulatory network topological units coordinating the genome-wide transcriptional response to glucose in Escherichia coli.</h1><h3 class=mdl-color-text--gray>BMC Microbiology&nbsp;2007</h3><h2 class=mdl-color-text--teal-500>Rosa María Gutierrez-Ríos, Julio A Freyre-Gonzalez, Osbaldo Resendis, Julio Collado-Vides, Milton Saier and Guillermo Gosset</h2><p style=font-size:13px;margin-top:.5em></p><p>Glucose is the preferred carbon and energy source for Escherichia coli. A complex regulatory network coordinates gene expression, transport and enzyme activities in response to the presence of this sugar. To determine the extent of the cellular response to glucose, we applied an approach combining global transcriptome and regulatory network analyses.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1186/1471-2180-7-53>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/17559662>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Robustness and evolvability in genetic regulatory networks.</h1><h3 class=mdl-color-text--gray>Journal of Theoretical Biology&nbsp;2006</h3><h2 class=mdl-color-text--teal-500>Maximino Aldana, Enrique Balleza, Stuart Kauffman and Osbaldo Resendiz</h2><p style=font-size:13px;margin-top:.5em></p><p>Living organisms are robust to a great variety of genetic changes. Gene regulation networks and metabolic pathways self-organize and reaccommodate to make the organism perform with stability and reliability under many point mutations, gene duplications and gene deletions. At the same time, living organisms are evolvable, which means that these kind of genetic perturbations can eventually make the organism acquire new functions and adapt to new environments. It is still an open problem to determine how robustness and evolvability blend together at the genetic level to produce stable organisms that yet can change and evolve. Here we address this problem by studying the robustness and evolvability of the attractor landscape of genetic regulatory network models under the process of gene duplication followed by divergence. We show that an intrinsic property of this kind of networks is that, after the divergence of the parent and duplicate genes, with a high probability the previous phenotypes, encoded in the attractor landscape of the network, are preserved and new ones might appear. The above is true in a variety of network topologies and even for the case of extreme divergence in which the duplicate gene bears almost no relation with its parent. Our results indicate that networks operating close to the so-called &ldquo;critical regime&rdquo; exhibit the maximum robustness and evolvability simultaneously.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1016/j.jtbi.2006.10.027>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/17188715>PubMed</a></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class=mdl-card__supporting-text><h1 class=mdl-color-text--primary>Modular analysis of the transcriptional regulatory network of E. coli.</h1><h3 class=mdl-color-text--gray>Trends in Genetics : TIG&nbsp;2005</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio, Julio A Freyre-González, Ricardo Menchaca-Méndez, Rosa M Gutiérrez-Ríos, Agustino Martínez-Antonio, Cristhian Avila-Sánchez and Julio Collado-Vides</h2><p style=font-size:13px;margin-top:.5em></p><p>The transcriptional network of Escherichia coli is currently the best-understood regulatory network of a single cell. Motivated by statistical evidence, suggesting a hierarchical modular architecture in this network, we identified eight modules with well-defined physiological functions. These modules were identified by a clustering approach, using the shortest path to trace regulatory relationships across genes in the network. We report the type (feed forward and bifan) and distribution of motifs between and within modules. Feed-forward motifs tend to be embedded within modules, whereas bi-fan motifs tend to link modules, supporting the notion of a hierarchical network with defined functional modules.</p></div><div class="mdl-card__actions mdl-card--border"><a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://doi.org/10.1016/j.tig.2004.11.010>Full Article</a>
<a class="mdl-button mdl-button--colored mdl-js-button mdl-js-ripple-effect" href=https://www.ncbi.nlm.nih.gov/pubmed/15680508>PubMed</a></div></div></div></div><div class=mdl-tabs__panel id=books><br><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class="mdl-card__supporting-text book"><h1 class=mdl-color-text--primary>Cancer: a complex disease</h1><h3 class=mdl-color-text--gray>December 2018</h3><h2 class=mdl-color-text--teal-500>Elena R. Alvarez-Buylla, Juan Carlos Balandran, Jose Luis Caldu-Primo, Jose Davila-Velderrain, Jennifer Enciso, Enrique Hernandez-Lemus, Lucia S. Lopez Castillo, Juan Carlos Martinez-Garcia, Nancy R. Mejia-Dominguez, Leticia R. Paiva, Rosana Pelayo, Osbaldo Resendis-Antonio and Octavio Valadez-Blanco</h2><p style=font-size:13px;margin-top:.5em></p><p><img src=../media/cancer_a_complex_disease_ebook.png alt></p><p><em>This is an EBook can
be <a href=http://scifunam.fisica.unam.mx/mir/copit/TS0017EN/TS0017EN.pdf>downloaded for free</a>.</em></p><p>The study of complex systems and their related phenomena has become a major research venue in the recent years and it is commonly regarded as an important part of the scientific revolution developing through the 21st century. The science of complexity is concerned with the laws of operation and evolution of systems formed by many locally interacting elements that produce collective order at spatiotemporal scales larger than that of the single constitutive elements. This new thinking, that explores formally the emergence of spontaneous higher order and feedback hierarchies, has been particularly successful in the biological sciences. One particular life-threatening disease in humans, overwhelmingly common in the modern world is cancer. It is regarded as a collection of phenomena involving anomalous cell growth caused by an underlying genetic instability with the potential to spread to other parts of the human body. In the present book, a group of well recognized specialists discuss new ideas about the disease. These authors coming from solid backgrounds in physics, mathematics, medicine, molecular and cell biology, genetics and anthropology have dedicated their time to write an authoritative free-available text published under the open access philosophy that hopefully would be in the front-line struggle against cancer, a complex disease.</p></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class="mdl-card__supporting-text book"><h1 class=mdl-color-text--primary>Quantitative Models for Microscopic to Macroscopic Biological Macromolecules and Tissues</h1><h3 class=mdl-color-text--gray>March 2018</h3><h2 class=mdl-color-text--teal-500>Luis Olivarez-Quiroz and Osbaldo Resendis-Antonio</h2><p style=font-size:13px;margin-top:.5em></p><p><img src=../media/quantitative_models.jpg alt></p><p>This book presents cutting-edge research on the use of physical and
mathematical formalisms to model and quantitatively analyze biological
phenomena ranging from microscopic to macroscopic systems. The systems
discussed in this compilation cover protein folding pathways, gene regulation
in prostate cancer, quorum sensing in bacteria to mathematical and physical
descriptions to analyze anomalous diffusion in patchy environments and the
physical mechanisms that drive active motion in large sets of particles, both
fundamental descriptions that can be applied to different phenomena in biology.</p></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class="mdl-card__supporting-text book"><h1 class=mdl-color-text--primary>Systems Biology and the Challenge of Deciphering the Metabolic Mechanisms Underlying Cancer</h1><h3 class=mdl-color-text--gray>November 2017</h3><h2 class=mdl-color-text--teal-500>Osbaldo Resendis-Antonio and Christian Diener</h2><p style=font-size:13px;margin-top:.5em></p><p><img src=../media/frontiers_ebook.jpg alt></p><p><em>This is an EBook compendium of the respective Frontiers Research Topic and can
be <a href=https://www.frontiersin.org/books/Systems_Biology_and_the_Challenge_of_Deciphering_the_MetabolicMechanisms_Underlying_Cancer/1387#nogo>downloaded for free</a>.</em></p><p>Since the discovery of the Warburg effect in the 1920s cancer has been tightly
associated with the genetic and metabolic state of the cell. One of the
hallmarks of cancer is the alteration of the cellular metabolism in order to
promote proliferation and undermine cellular defense mechanisms such as
apoptosis or detection by the immune system. However, the strategies by which
this is achieved in different cancers and sometimes even in different patients
of the same cancer is very heterogeneous, which hinders the design of general
treatment options. Recently, there has been an ongoing effort to study this
phenomenon on a genomic scale in order to understand the causality underlying
the disease. Hence, current “omics” technologies have contributed to identify
and monitor different biological pieces at different biological levels, such as
genes, proteins or metabolites. These technological capacities have provided us
with vast amounts of clinical data where a single patient may often give rise
to various tissue samples, each of them being characterized in detail by
genomescale data on the sequence, expression, proteome and metabolome level.
Data with such detail poses the imminent problem of extracting meaningful
interpretations and translating them into specific treatment options. To this
purpose, Systems Biology provides a set of promising computational tools in
order to decipher the mechanisms driving a healthy cell’s metabolism into a
cancerous one. However, this enterprise requires bridging the gap between large
data resources, mathematical analysis and modeling specifically designed to
work with the available data. This is by no means trivial and requires high
levels of communication and adaptation between the experimental and theoretical
side of research.</p></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class="mdl-card__supporting-text book"><h1 class=mdl-color-text--primary>Encyclopedia of Systems Biology</h1><h3 class=mdl-color-text--gray>June 2013</h3><h2 class=mdl-color-text--teal-500>Werner Dubitzky, Olaf Wolkenhauer, Kwang-Hyun Cho and Hiroki Yokota</h2><p style=font-size:13px;margin-top:.5em></p><p><img src=../media/encyclopedia.jpg alt></p><p>The Encyclopedia of Systems Biology is conceived as a comprehensive reference work covering all aspects of systems biology, in particular the investigation of living matter involving a tight coupling of biological experimentation, mathematical modeling and computational analysis and simulation. The main goal of the Encyclopedia is to provide a complete reference of established knowledge in systems biology – a ‘one-stop shop’ for someone seeking information on key concepts of systems biology. As a result, the Encyclopedia comprises a broad range of topics relevant in the context of systems biology. The audience targeted by the Encyclopedia includes researchers, developers, teachers, students and practitioners who are interested or working in the field of systems biology. Keeping in mind the varying needs of the potential readership, we have structured and presented the content in a way that is accessible to readers from wide range of backgrounds. In contrast to encyclopedic online resources, which often rely on the general public to author their content, a key consideration in the development of the Encyclopedia of Systems Biology was to have subject matter experts define the concepts and subjects of systems biology.</p></div></div></div><div class="mdl-cell mdl-cell--12-col mdl-cell--4-col-phone mdl-cell--8-col-tablet"><div class="mdl-card mdl-shadow--4dp"><div class="mdl-card__supporting-text book"><h1 class=mdl-color-text--primary>Symbiotic Endophytes</h1><h3 class=mdl-color-text--gray>January 2013</h3><h2 class=mdl-color-text--teal-500>Ricardo Aroca</h2><p style=font-size:13px;margin-top:.5em></p><p><img src=../media/endophytes.jpg alt></p><p>This Soil Biology volume examines our current understanding of the mechanisms involved in the beneficial effects transferred to plants by endophytes such as rhizobial, actinorhizal, arbuscular mycorrhizal symbionts and yeasts.
Topics presented include how symbiosis starts on the molecular level; chemical signaling in mycorrhizal symbiosis; genomic and functional diversity of endophytes; nitrogen fixation; nutrient uptake and cycling; as well as plant protection against various stress conditions. Further, the use of beneficial microorganisms as biopesticides is discussed, particularly the application of Plant Growth Promoter Rhizobacteria (PGPR) in agriculture with the aim to increase yields.</p></div></div></div></div></div></div></main></div><script src=https://storage.googleapis.com/code.getmdl.io/1.3.0/material.min.js></script></body></html>